src/webcomponents/user/user-password-change.js

```diff
@@ -16,55 +17,101 @@ export default class UserPasswordChange extends LitElement {
 
     static get properties() {
         return {
+            userId: {
+                type: String,
+            },
             opencgaSession: {
                 type: Object
             },
+            organizationId: {
+                type: String,
+            },
+            displayConfig: {
+                type: Object
+            },
         };
     }
 
     #init() {
-        this.updateParams = {};
-        this.config = this.getDefaultConfig();
+        this._password = {};
+        this.displayConfigDefault = {
+            style: "margin: 10px",
+            titleWidth: 3,
+            titleStyle: "color: var(--main-bg-color);margin-bottom:16px;font-weight:bold;",
+            defaultLayout: "horizontal",
+            buttonOkText: "Change password",
+        };
+        this._config = this.getDefaultConfig();
+    }
+
+    #setLoading(value) {
+        this.isLoading = value;
+        this.requestUpdate();
+    }
+
+    #initOriginalObjects() {
+        this._password = {};
+        this._config = this.getDefaultConfig();
+        this.requestUpdate();
+    }
+
+    update(changedProperties) {
+        if (changedProperties.has("displayConfig")) {
+            this.displayConfig = {
+                ...this.displayConfigDefault,
+                ...this.displayConfig
+            };
```

Comment:
I would recommend not to change the original `this.displayConfig` object. Instead, merge both objects in the `getDefaultConfig` method:

```
getDefaultConfig() {
    return {
        displayConfig: {
            ...this.displayConfigDefault,
            ...this.displayConfig,
        },
        // ...
    };
}
```

Reply:
I completely agree. However, we'll need to discuss this further and ensure the changes are propagated across IVA for consistency. I coded it this way because it follows the agreed-upon template used in all views and creates. I'll leave it as it is for now until we reach an agreement.